packages/fim-client/src/al-fim-client.ts

```diff
@@ -4,12 +4,9 @@ import {
 } from '@al/core';
 import {
     AlFimConfiguration,
-    AlFimConfigurationPayload
+    fimPathType
```

Comment:
I think you can just remove this import altogether now, it looks like it's being unused in this particular file. It's already being exported in the `export * from './types'` in your `index.ts`